_translations/index_fi.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: signed
+title: An open letter in support of Richard M. Stallman
+description: An open letter in support of Richard Matthew Stallman being reinstated by the Free Software Foundation
+image: /assets/social-media-preview.png
+locale: fi_FI
+twitter:
+  card: summary_large_image
+---
+
+2021-03-23
+
+Richard M. Stallman, joka myös tunnetaan nimimerkillä RMS on ollut vapaan ohjelmistoliikkeen vapaan ohjelmistoliikkeen johtava voima useamman vuosikymmenen ajan ja on tukenut liikettä työpanoksellaan, johon sisältyy GNU-käyttöjärjestelmä ja Emacs-tekstieditori.
+Hiljattain Richard Stallman on joutunut katalan hyökkäyksen kohteeksi netissä, jonka tavoitteena on hänen erottaminen Free Software Foundationin johtokunnasta. Olemme nähneet samankaltaisen järjestäytyneen hyökkäyksen tapahtuneen muille merkittäville vapaan ohjelmiston aktivisteille ja ohjelmoijille. Emme tule katsomaan vierestä, kun liikkeemme johtohahmo joutuu hyökkäyksen kohteeksi.
```

Comment:
> Hiljattain Richard Stallman on joutunut **katalan** hyökkäyksen kohteeksi netissä, jonka tavoitteena on hänen erottaminen Free Software Foundationin johtokunnasta.

Machine translates into:

_Recently, Richard Stallman has been the subject of a **Catalan** attack on the Internet with the aim of removing him from the board of the Free Software Foundation._

Is the _katalan_ here really correct? The machine translation sounds like it's the fault of Catalans, which is not true. I'm not Finnish, so I don't know.

Reply:
Machine translation can be sometimes a bit tricky. Katala translates quite literally into "vicious". If you're talking about the Catalans, you'd want to use the adjective "katalaani".

Reply:
Ok, I see. I will take your word for it. We can always fix it in another PR if necessary.